Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -366,6 +369,14 @@ class IncidentType(Enum):
 
 """ HELPER FUNCTIONS """
 
+def is_fetch_events():
```

Comment:
seems unnecessary to declare a method for a single line of comparison.
consider making the comparison instead of calling a method.